modules/api-compatibility-guidelines.adoc

```diff
@@ -13,6 +13,7 @@ Red Hat recommends that application developers adopt the following principles in
 * Do not design applications that rely on configuration files provided by system packages or other components. These files can change between versions unless the upstream community is explicitly committed to preserving them. Where appropriate, depend on any Red Hat provided interface abstraction over those configuration files in order to maintain forward compatibility. Direct file system modification of configuration files is discouraged, and users are strongly encouraged to integrate with an Operator provided API where available to avoid dual-writer conflicts.
 * Do not depend on API fields prefixed with `unsupported<FieldName>` or annotations that are not explicitly mentioned in product documentation.
 * Do not depend on components with shorter compatibility guarantees than your application.
+* Do not perform direct storage operations on the etcd server, all etcd access must be performed via the api-server or through documented backup/restore procedures.
```

Comment:
```suggestion
* Do not perform direct storage operations on the etcd server, all etcd access must be performed via the api-server or through documented backup or restore procedures.
```
IBM Style Guide: Avoid using forward slashes (/) and backslashes (\) in most text and in dates.